<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QMainWindow" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>550</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="intervalCheckBox">
          <property name="text">
           <string>Hours and Minutes</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSpinBox" name="spinBoxHour">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Hours</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSpinBox" name="spinBoxMinute">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Minutes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="scheduledCheckBox">
          <property name="text">
           <string>Set a time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="setUpButton">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButon">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="info">
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>intervalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxHour</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intervalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intervalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxMinute</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intervalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scheduledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
